apps/ledger-live-desktop/src/renderer/modals/Send/steps/Confirmation/NodeError/TechnicalErrorSection.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import { Flex, Text, Icons } from "@ledgerhq/react-ui";
+import { useTranslation } from "react-i18next";
+import styled from "styled-components";
+
+type Props = {
+  error: Error;
+  onSaveLogs: () => void;
+  onCopyError: () => void;
+};
+
+const InteractFlex = styled(Flex)`
+  &:hover {
+    cursor: pointer;
+  }
+  padding: 8px;
+  background-color: ${({ theme }) => theme.colors.palette.opacityDefault.c05};
+  border-radius: 8px;
+  column-gap: 8px;
+`;
+
+const TechnicalErrorSection = ({ error, onSaveLogs, onCopyError }: Props) => {
+  const { t } = useTranslation();
+  return (
+    <Flex
+      flex={0.5}
+      alignItems="flex-start"
+      flexDirection="column"
+      bg="opacityDefault.c05"
+      borderRadius={8}
+      justifyContent="space-between"
+      p={2}
+      overflow="hidden"
```

Comment:
In which case this wouldn't be needed anymore.

Reply:
removed